v2/getting-started/composer.md

```diff
@@ -1,6 +1,51 @@
 # Composer: the what, why, and how
 
-@todo
+## What is Composer?
+**[Composer](https://getcomposer.org/)** is the de facto dependency management tool for PHP. Composer allows PHP developers to easily build standalone distributable libraries that can be shared and integrated by others. This is possible in part by the PHP Framework Interoperability Group (FIG) and [PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading of class files. 
 
-* Port contents of https://glamanate.com/blog/managing-your-drupal-project-composer
-* And https://docs.google.com/presentation/d/1PK9q2dBkGHfyEO76bEVpqS61wTgA0LGbru2PECiwUnk/edit?usp=sharing
\ No newline at end of file
+[PEAR](http://pear.php.net/) is another management tool, however it installs PHP libraries in a global context via `include` and `require`. Composer supports both per-project and global usages. PEAR also does not handle dependency management.
+
+> Dependency management is not a new concept and not alone to PHP. NPM for NodeJS, Bower for front end libraries, Bundler/Gems for Ruby, PIP for Python, Maven for Java and so forth. 
+
+## Why does Drupal Commerce need it?
+
+Drupal Commerce utilizes various [libraries and dependencies](v2/building-blocks/index.md).
+
+Drupal Commerce 2 was started first as shared libraries that solve  the common problems of selling online — regardless the platform you are using. This covers addressing, internationalization, and taxes. These libraries are then integrated into Drupal.
```

Comment:
We also need to emphasize our reliance on Drush Make in D7 and how Composer is a continuation of that. Commerce emphasizes best practises.